prow/jobs/test-infra/prow-addons-ctrl-manager.yaml

```diff
@@ -0,0 +1,45 @@
+job_template: &job_template
+  skip_report: false
+  decorate: true
+  path_alias: github.com/kyma-project/kyma
+  max_concurrency: 10
+  spec:
+    containers:
+    - image: eu.gcr.io/kyma-project/prow/test-infra/buildpack-golang:v20181119-afd3fbd
+      securityContext:
+        privileged: true
+      command:
+      - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/build.sh"
+      args:
+      - "/home/prow/go/src/github.com/kyma-project/test-infra/development/prow-addons-ctrl-manager"
+      resources:
+        requests:
+          memory: 1.5Gi
+          cpu: 0.8
+
+job_labels_template: &job_labels_template
+  preset-dind-enabled: "true"
+  preset-sa-gcr-push: "true"
+  preset-docker-push-repository-test-infra: "true"
+
+presubmits: # runs on PRs
```

Comment:
where can I find tests for this yaml file :) ?

Reply:
🤦‍♂️